LICENSE-3rdparty.csv

```diff
@@ -65,6 +65,7 @@ core,"github.com/gregjones/httpcache/diskcache",MIT
 core,"github.com/hako/durafmt",MIT
 core,"github.com/hashicorp/golang-lru",MPL-2.0
 core,"github.com/hashicorp/golang-lru/simplelru",MPL-2.0
+core,"github.com/heptiolabs/healthcheck",Apache-2.0
 core,"github.com/imdario/mergo",NewBSD
```

Comment:
Please use SPDX identifiers for the license field.

Reply:
2 additional notes:
* The LICENSE file should contain the Apache 2 license template.
* We have a default NOTICE file that must be used.

Reply:
Hi @jeremy-lq
I created a dedicated PR for fixing the Licence issue: https://github.com/DataDog/datadog-operator/pull/53